sass/main.scss

```diff
@@ -1147,6 +1148,22 @@ footer {
             font-size: 3em;
             padding: 0px 5px;
         }
+
+        a {
+            display: inline-block;
+            transition: 0.2s;
+
+            i {
+                transition: 0.2s;
+            }
+            &:hover {
```

Comment:
nit: `&:` styles should go above any other nested CSS, so this should go above the `i` above